src/go/configgenerator/listener_generator.go

```diff
@@ -182,7 +191,28 @@ func MakeListener(serviceInfo *sc.ServiceInfo) (*v2pb.Listener, error) {
 		return nil, err
 	}
 
+	filterChain := &listenerpb.FilterChain{
+		Filters: []*listenerpb.Filter{
+			{
+				Name:       util.HTTPConnectionManager,
+				ConfigType: &listenerpb.Filter_TypedConfig{TypedConfig: httpFilterConfig},
+			},
+		},
+	}
+
+	listenerName := "http_listener"
```

Comment:
Maybe move to util

Reply:
I think the principal is putting some common strings in util, so that it can be used by multi files. Here this listener name is specific used here only.